lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -29,28 +29,18 @@
 
 LOG_MODULE_REGISTER(nrf_modem, CONFIG_NRF_MODEM_LIB_LOG_LEVEL);
 
-struct mem_diagnostic_info {
-	uint32_t failed_allocs;
-};
-
 struct sleeping_thread {
 	sys_snode_t node;
 	struct k_sem sem;
 };
 
-/* Shared memory heap
- * This heap is not initialized with the K_HEAP macro because
- * it should be initialized in the shared memory area reserved by
- * the Partition Manager. This happens in `nrf_modem_os_init()`.
- */
-static struct k_heap shmem_heap;
+/* Heaps, extern in diag.c */
 
-/* Library heap, defined here in application RAM */
-static K_HEAP_DEFINE(library_heap, CONFIG_NRF_MODEM_LIB_HEAP_SIZE);
-
-/* Store information about failed allocations */
-static struct mem_diagnostic_info shmem_diag;
-static struct mem_diagnostic_info heap_diag;
+/* Shared memory heap */
+struct k_heap shmem_heap;
+/* Library heap */
+struct k_heap library_heap;
```

Comment:
When making these not `static`, should they be prefixed with `nrf_modem_lib`?

Reply:
Ok